### PR TITLE
Anoosha-GenEpio

### PR DESCRIPTION
**Pull request by Anoosha-Sehar**
GRDI terms

### CODE REVIEW

File: src/ontology/genepio-edit.owl [hunk 1]
```diff
@@ -14413,6 +14442,15 @@ AnnotationAssertion(dc11:date obo:GENEPIO_0002229 "2019-04-17T22:47:51Z"^^xsd:da
 AnnotationAssertion(rdfs:label obo:GENEPIO_0002229 "PulseNet:UploadDate"@en)
 ClassAssertion(obo:GENEPIO_0002185 obo:GENEPIO_0002229)
 
+# Individual: obo:GENEPIO_0004434 (University of Manitoba (UM))
+
+AnnotationAssertion(obo:IAO_0000115 obo:GENEPIO_0004434 "A canadian public research university in the province of Manitoba."@en)
```
Comment (ddooley) [inline src/ontology/genepio-edit.owl:14447]: Small thing: Canadian needs a a capital C!

**Review by ddooley — CHANGES_REQUESTED** [review]

File: src/ontology/genepio-edit.owl (hunk 1 above)
Comment (Anoosha-Sehar) [reply src/ontology/genepio-edit.owl]: Just corrected that and pushed the changes.

File: src/ontology/genepio-edit.owl [hunk 2]
```diff
@@ -11689,7 +11692,10 @@ SubClassOf(obo:GENEPIO_0002124 obo:NCIT_C43167)
 # Class: obo:GENEPIO_0002125 (hand scoop)
 
 AnnotationAssertion(obo:IAO_0000114 obo:GENEPIO_0002125 obo:IAO_0000122)
+AnnotationAssertion(obo:IAO_0000115 obo:GENEPIO_0002125 "A shovel-like utensil that has a deep curved dish and a short handle and is used for digging into a soft substance for lifting out a portion."@en)
 AnnotationAssertion(obo:IAO_0000117 obo:GENEPIO_0002125 "Emma Griffiths"@en)
+AnnotationAssertion(dc:contributor obo:GENEPIO_0002125 <https://orcid.org/0000-0002-1107-9135>)
+AnnotationAssertion(dc:date obo:GENEPIO_0002125 "2023-06-21T13:54:32Z"^^xsd:dateTime)
```
[the remark this answers is not in the document]
Comment (Anoosha-Sehar) [reply src/ontology/genepio-edit.owl:11698]: Regarding this one, I checked but the terms in my PR already have 'dc: terms', not sure which term is this one?

File: src/ontology/genepio-edit.owl [hunk 3]
```diff
@@ -9647,8 +9650,8 @@ AnnotationAssertion(obo:IAO_0000115 obo:GENEPIO_0001839 "PulseNet, a national an
 AnnotationAssertion(obo:IAO_0000118 obo:GENEPIO_0001839 "Pulse Net International BioSample Metadata"@en)
 AnnotationAssertion(oboInOwl:hasDbXref obo:GENEPIO_0001839 "PulseNet")
 AnnotationAssertion(rdfs:label obo:GENEPIO_0001839 "PulseNet submission standard"@en)
-AnnotationAssertion(rdfs:seeAlso obo:GENEPIO_0001839 "https://www.nml-lnm.gc.ca/Pulsenet/index-eng.htm"@en)
 AnnotationAssertion(rdfs:seeAlso obo:GENEPIO_0001839 "https://www.cdc.gov/pulsenet/pathogens/index.html"^^xsd:anyURI)
+AnnotationAssertion(rdfs:seeAlso obo:GENEPIO_0001839 "https://www.nml-lnm.gc.ca/Pulsenet/index-eng.htm"@en)
```
[the remark this answers is not in the document]
Comment (Anoosha-Sehar) [reply src/ontology/genepio-edit.owl]: Okay, but these terms are not part of my pull request. But I have anyways corrected the annotations in this term 'PulseNet submission standard 'using the IRI editor. I just pushed these changes. Let me know if you want me to correct anything else.

**Review by ddooley — APPROVED** [review]

**Pull request merged**